lib/polyfill/mediakeys.js

```diff
@@ -52,12 +52,15 @@ shaka.polyfill.MediaKeys.install = function() {
     shaka.log.info('Using native EME as-is.');
   } else if (HTMLMediaElement.prototype.webkitGenerateKeyRequest) {
     shaka.log.info('Using prefixed EME v0.1b.');
+    /** @suppress {missingRequire} */
```

Comment:
I think the compiler does not like the way the namespace is formed. These are the errors:

```
/home/ahochhaus/shaka-player2/lib/polyfill/mediakeys.js:55: ERROR - missing require: 'shaka.polyfill.PatchedMediaKeys'
    shaka.polyfill.PatchedMediaKeys.v01b.install();
    ^

/home/ahochhaus/shaka-player2/lib/polyfill/mediakeys.js:58: ERROR - missing require: 'shaka.polyfill.PatchedMediaKeys'
    shaka.polyfill.PatchedMediaKeys.v20140218.install();
    ^

/home/ahochhaus/shaka-player2/lib/polyfill/mediakeys.js:61: ERROR - missing require: 'shaka.polyfill.PatchedMediaKeys'
    shaka.polyfill.PatchedMediaKeys.nop.install();
    ^

3 error(s), 0 warning(s)
```

I lowered the "p" in "patchedMediaKeys" in the next patchset which allows me to omit the suppress while keeping the compiler happy. If you prefer a different work around please let me know.